schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -275,6 +275,18 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-30/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_python2Packages" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-30/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_runbooks" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-10-01/Microsoft.Media.json#/resourceDefinitions/mediaServices" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-07-01/Microsoft.Media.json#/resourceDefinitions/mediaServices" },
```

Comment:
>mediaServices [](start = 126, length = 13)

please use consistent case here and in the schema file.

Reply:
Fixed.  I removed the duration format mentioned above and standardized the casing of the definitions in the deployment templates, schema, and the associated tests.